Doc/whatsnew/3.13.rst

```diff
@@ -144,6 +144,15 @@ pathlib
   :meth:`~pathlib.Path.is_dir`.
   (Contributed by Barney Gale in :gh:`77609` and :gh:`105793`.)
 
+sqlite3
+-------
+
+* Passing more than one positional argument to :func:`sqlite3.connect` is
+  deprecated. Parameters *timeout*, *detect_types*, *isolation_level*,
+  *check_same_thread*, *factory*, *cached_statements*, and *uri* is deprecated.
+  They will become keyword-only parameters in Python 3.15.
```

Comment:
Contradiction. Parameters is deprecated. They will become keyword-only parameters.

Also, it helps that all these parameters specify optional arguments, so you can say "other parameters" or "parameters which specify optional arguments" instead of enumerating their names.

For example:
```suggestion
  deprecated. All parameters which specify optional arguments
  will become keyword-only parameters in Python 3.15.
```

Reply:
The original sentence I wrote said _"Positional use of parameters ..."_, but then I changed the beginning of the sentence before committing 😆

Reply:
Anyway, I guess it is ok to say "optional arguments" here.